cli/maintainer/src/daemon.rs

```diff
@@ -104,7 +104,7 @@ impl MaintenanceMetrics {
                         .with_label("operation", "StakeDeposit".to_string()),
                     Metric::new(self.transactions_update_exchange_rate)
                         .with_label("operation", "UpdateExchangeRate".to_string()),
-                    Metric::new(self.transactions_withdraw_inactive_stake)
+                    Metric::new(self.transactions_update_stake_account_balance)
                         .with_label("operation", "WithdrawInactiveStake".to_string()),
```

Comment:
We should update the label as well, this is what gets exposed in the end.
```suggestion
                        .with_label("operation", "UpdateStakeAccountBalance".to_string()),
```

Reply:
sharp eye, fixed